styles/markdown.scss

```diff
@@ -0,0 +1,39 @@
+// Default styles to apply to markdown generated content
+
+@import "variables.scss";
+
+.rendered-markdown {
+  margin-bottom: var(--spacing-xl);
+
+  h1 {
+    margin-top: var(--spacing-md);
+  }
```

Comment:
I find that it looks too balanced for most headings. As in the margin on top is equal to the margin below. It should be a larger space above, smaller below. It is subtle but it improves readability. First screenshot is this PR, second is Figma for reference

<img width="826" alt="Screenshot 2024-04-09 at 9 30 34 AM" src="https://github.com/bocoup/p5.js-website/assets/10382506/54b91011-b6e1-4a8b-ad2f-0e7ab8cc2ef2">
<img width="761" alt="Screenshot 2024-04-09 at 9 38 12 AM" src="https://github.com/bocoup/p5.js-website/assets/10382506/b5fae109-424e-4462-9ba8-80ba9e88f5b2">